recipes/gbmunge/makefile.patch

```diff
@@ -0,0 +1,30 @@
+--- Makefile.old	2019-04-24 17:17:58.618580369 -0500
++++ Makefile	2019-04-24 17:23:40.137613637 -0500
+@@ -1,5 +1,13 @@
+-all:
++PREFIX :=
++EXECUTABLE := gbmunge 
++
++all: $(EXECUTABLE)
++
++$(EXECUTABLE):
+ 	$(MAKE) -C src
+ 
+ clean:
+ 	$(MAKE) -C src clean
++
++install:
++	cp src/$(EXECUTABLE) $(PREFIX)/bin/
+--- Make.inc.old	2019-04-24 17:15:43.499565380 -0500
++++ Make.inc	2019-04-24 17:16:10.072568329 -0500
+@@ -10,10 +10,8 @@
+ 
+ ifeq ($(USECLANG),1)
+ USEGCC = 0
+-CC = clang
+ endif
+ 
+ ifeq ($(USEGCC),1)
+-CC = gcc
```

Comment:
You can achieve the same by calling

```
make CC=$CC
```

Variables set on the `make` commandline override variables set inside of the `makefile` (even if there is `CFLAGS += -bla` going on!).